client/src/components/Pages/Structure/Builder/Builder.tsx

```diff
@@ -183,162 +244,193 @@ const Builder: React.FC = () => {
     }
   };
 
-  /**
-   * Render nomenclature preview footer underneath builder interface
-   * @returns completed footer element
-   */
-  const renderFooterBar = () => (
-    <div className="hr-section">
-      {fusion.structural_elements
-        ?.filter(
-          (element: ClientElementUnion) =>
-            Boolean(element) && element.nomenclature
-        )
-        .map((element: ClientElementUnion, index: number) => (
-          <div key={element.element_id}>{`${index ? "::" : ""}${
-            element.nomenclature
-          }`}</div>
-        ))}
-    </div>
+  const nomenclatureContent = fusion.structural_elements
+    .filter(
+      (element: ClientElementUnion) => Boolean(element) && element.nomenclature
+    )
+    .map(
+      (element: ClientElementUnion, index: number) =>
+        `${index ? "::" : ""}${element.nomenclature}`
+    );
+
+  const nomenclatureElement = (
+    <Box
```

Comment:
This might just be my preference, but I think it might be good to give this box a min-height so that it renders at the same size regardless of whether there's a nomenclature value available. That way everything else isn't pushed down once you enter a complete component.

Reply:
I like your thinking, lol! So I actually had the same thought but was having a terrible time trying to actually get this to play nicely and work fine whether there was a label in here or not. But now that I think about it, I don't think I tried a min-height. Let me try that